cosmosdb/cassandra/ps-cassandra-ru-get.ps1

```diff
@@ -0,0 +1,20 @@
+# Get RU for an Azure Cosmos Cassandra API keyspace or table
+$apiVersion = "2015-04-08"
+$resourceGroupName = "myResourceGroup"
+$accountName = "mycosmosaccount"
+$keyspaceName = "keyspace1"
+$tableName = "table1"
+$keyspaceThroughputResourceType = "Microsoft.DocumentDb/databaseAccounts/apis/keyspaces/settings"
+$tableThroughputResourceType = "Microsoft.DocumentDb/databaseAccounts/apis/keyspaces/tables/settings"
+$keyspaceThroughputResourceName = $accountName + "/cassandra/" + $keyspaceName + "/throughput"
+$tableThroughputResourceName = $accountName + "/cassandra/" + $keyspaceName + "/" + $tableName + "/throughput"
+
+# Get the throughput for a keyspace (returns RU/s or error if not set)
```

Comment:
@sptrammer Is there a template for this suggestion? There are several of these calls in a row, so is it just checking for NULL? Would love to see an example if you have a moment.

Reply:
No template and the check does not have to be in code. It can just be a comment that says what the "error if not set" response is. I assume that it's some variety of 4xx response but the code/status message should be spelled out.